adapters/Unity/UnityAdapter/GADMAdapterUnityRewardedAd.m

```diff
@@ -83,100 +63,61 @@ - (void)requestRewardedAd {
 
   if (![UnityAds isSupported]) {
     NSString *description =
-        [[NSString alloc] initWithFormat:@"%@ is not supported for this device.",
-                                         NSStringFromClass([UnityAds class])];
-
+    [[NSString alloc] initWithFormat:@"%@ is not supported for this device.",
+     NSStringFromClass([UnityAds class])];
     if (_adLoadCompletionHandler) {
-      NSError *error = GADMAdapterUnityErrorWithCodeAndDescription(
-          GADMAdapterUnityErrorDeviceNotSupported, description);
+      NSError *error = GADMAdapterUnityErrorWithCodeAndDescription(GADMAdapterUnityErrorDeviceNotSupported, description);
       _adLoadCompletionHandler(nil, error);
       _adLoadCompletionHandler = nil;
     }
     return;
   }
```

Comment:
I think we can remove this part since `initializeWithGameID` already checks this.